.eslintrc

```diff
@@ -74,26 +74,26 @@
     "no-dupe-args": "error",              // disallow duplicate arguments in function definitions
     "no-dupe-keys": "error",              // disallow duplicate keys in object literals
     "no-duplicate-case": "error",         // disallow duplicate case labels
-    "no-empty": 0,                        // disallow empty block statements
+    "no-empty": "error",                  // disallow empty block statements
```

Comment:
this doesn't prevent empty function definitions or empty object definitions - which is good